src/IB/FEMechanicsBase.cpp

```diff
@@ -1272,6 +1601,30 @@ FEMechanicsBase::getFromInput(const Pointer<Database>& db, bool /*is_from_restar
     d_libmesh_partitioner_type =
         string_to_enum<LibmeshPartitionerType>(db->getStringWithDefault("libmesh_partitioner_type", "LIBMESH_DEFAULT"));
 
+    // Restart settings.
+    if (db->isString("libmesh_restart_file_extension"))
+    {
+        d_libmesh_restart_file_extension = db->getString("libmesh_restart_file_extension");
+    }
+    else
+    {
+        d_libmesh_restart_file_extension = "xdr";
+    }
+
+    // FEProjector settings.
+    if (db->keyExists("FEProjector"))
+    {
+        d_fe_projector_db = db->getDatabase("FEProjector");
+    }
+    else
+    {
+        d_fe_projector_db = new InputDatabase("FEProjector");
+        if (db->keyExists("num_fischer_vectors"))
+        {
+            d_fe_projector_db->putInteger("num_fischer_vectors", db->getInteger("num_fischer_vectors"));
+        }
```

Comment:
```suggestion
```
I'd skip this  - if the user wants a certain number of fischer vectors they should supply an `FEProjector` database.